packages/@react-spectrum/autocomplete/chromatic/SearchAutocomplete.stories.tsx

```diff
@@ -79,7 +79,7 @@ let items = [
 export const Template: StoryFn<SpectrumSearchAutocompleteProps<object>> = (args) => (
   <Grid columns={repeat(4, '1fr')} autoFlow="row" gap="size-200">
     {combinations.map(c => {
-      let key = Object.keys(c).map(k => shortName(k, c[k])).join(' ');
+      let key = Object.keys(c).reverse().map(k => shortName(k, c[k])).join(' ');
```

Comment:
what caused this to change? You mentioned mergeProps, but what changed there? Seems like it could be potentially problematic for other usages too?

Reply:
mergeProps now merges in the opposite direction https://github.com/adobe/react-spectrum/pull/7657/files which just results in the items being added into the object in the opposite order for this particular use case, the values in those keys remains the same

relying on the order of Object.keys isn't something one should do anyways, it just happened that we do it for labels in chromatic

it'd be better to sort them in some way, but we can handle that as a separate pr, I wanted to know if we were actually breaking anything in this PR